src/component/MainContent.jsx

```diff
@@ -2,25 +2,58 @@ import React, {useState} from 'react';
 import classes from './MainContent.module.css'
 import Input from './UI/Input'
 import ButtonEquals from "./ButtonEquals";
-import ResultTable from "./ResultTable"
+import ResultTable1 from "./ResultTable1";
+import ArrowSortDown from "./UI/ArrowSortDown";
+import ArrowSortUp from "./UI/ArrowSortUp";
+import ArrowSortUpDown from "./UI/ArrowSortUpDown";
+
 
 const MainContent = () => {
   const [results, setResults] = useState([])
    const [num, setNum] = useState()
    const [num1, setNum1] = useState()
+   const [order, setOrder] = useState('ASC')
+   const[field, setField] = useState('')
+
+
+   const sortBy = (sort) => {
+     if (order === 'ASC') {
+        const sorted = [...results].sort((a,b) =>
+           a[sort] > b[sort] ? 1 : -1
+        );
+        setResults(sorted);
+        setOrder('DSC');
+     }
+      if (order === 'DSC') {
+         const sorted = [...results].sort((a,b) =>
+            a[sort] < b[sort] ? 1 : -1
+         );
+         setResults(sorted);
+         setOrder('ASC');
+      }
```

Comment:
do you see repeated logic ? it should be optimized, read this:
https://www.itexico.com/blog/software-development-kiss-yagni-dry-3-principles-to-simplify-your-life